doc/RecordingSearch.xml

```diff
@@ -1486,6 +1547,107 @@ http://www.onvif.org/ver10/tptz/ZoomSpaces/PositionGenericSpace
       <para>The definition is available via the link &lt;<link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://www.onvif.org/specs/srv/rsrch/ONVIF-RecordingSearch-Service-Spec-v1606.pdf">https://www.onvif.org/specs/srv/rsrch/ONVIF-RecordingSearch-Service-Spec-v1606.pdf</link>&gt;.</para>
     </section>
   </appendix>
+  <appendix xml:id="_refSegmentExport">
```

Comment:
Shouldn't be this section be defined in Recording Control Service?

Reply:
Half the calls are in search and the other half in control so we were a bit unclear on where would be the best place for it. Open to switching it

Reply:
Moved from Search to Recording Control